cartography/intel/gcp/compute.py

```diff
@@ -694,6 +808,155 @@ def load_gcp_forwarding_rules(neo4j_session: neo4j.Session, fwd_rules: List[Dict
             _attach_fwd_rule_to_vpc(neo4j_session, fwd, gcp_update_tag)
 
 
+@timeit
+def load_gcp_vpn_gateways(neo4j_session: neo4j.Session, vpngw_list: Dict, gcp_update_tag: int) -> None:
```

Comment:
[non-blocker] What kind of perf are you seeing with this job? It could be worth refactoring to do an `unwind .. merge`, but I won't block on this since I'm working on something to make the writes more standardized - will be trying in the coming weeks.

Reply:
I actually tried to do that but decided against it cause I found it hard to do it that way and then individually check for each element to call separately the `_attach_vpn_gateway_to_fwd_rules` method. Do you have an example where `UNWIND` is used along with an `_attach` method?

Reply:
@achantavy thoughts on this?

Reply:
The way you have it is fine; we'll fix it more broadly later.